ruleset.xml

```diff
@@ -1,5 +1,137 @@
+        <exclude-pattern>*typo3/sysext/core/Classes/Utility/DebugUtility.php</exclude-pattern>
+    </rule>
+
+    <!-- Files -->
+    <rule ref="Generic.Files.OneClassPerFile">
+        <exclude-pattern>*typo3/sysext/core/Migrations/Code/LegacyClassesForIde.php</exclude-pattern>
+    </rule>
+    <rule ref="Generic.Files.OneInterfacePerFile" />
+    <rule ref="Generic.Files.ByteOrderMark" />
+    <rule ref="Generic.Files.LineEndings" />
+    <rule ref="Generic.Files.LineLength">
+        <properties>
+            <property name="lineLimit" value="130"/>
+            <property name="absoluteLineLimit" value="200"/>
+        </properties>
+    </rule>
+    <rule ref="../TYPO3SniffPool/Sniffs/Files/IncludingFileSniff.php" />
+    <rule ref="../TYPO3SniffPool/Sniffs/Files/FilenameSniff.php" />
+    <rule ref="../TYPO3SniffPool/Sniffs/Files/FileExtensionSniff.php" />
+    <rule ref="Zend.Files.ClosingTag" />
+
+    <!-- Formatting -->
+    <rule ref="Generic.Formatting.DisallowMultipleStatements" />
+
+    <!-- Functions -->
+    <rule ref="Generic.Functions.FunctionCallArgumentSpacing" />
+    <rule ref="Generic.Functions.OpeningFunctionBraceKernighanRitchie" />
+
+    <!-- Loops -->
+
+    <!-- NamingConventions -->
+    <rule ref="../TYPO3SniffPool/Sniffs/NamingConventions/ValidVariableNameSniff.php" />
+    <rule ref="Generic.NamingConventions.ConstructorName" />
+    <rule ref="../TYPO3SniffPool/Sniffs/NamingConventions/ValidFunctionNameSniff.php" />
+
+    <!-- Objects -->
+
+    <!-- Operators -->
+
+    <!-- PHP -->
+    <rule ref="Generic.PHP.CharacterBeforePHPOpeningTag" />
+    <rule ref="Generic.PHP.DisallowShortOpenTag" />
+    <rule ref="Squiz.PHP.NonExecutableCode" />
+    <rule ref="../TYPO3SniffPool/Sniffs/PHP/DisallowMultiplePHPTagsSniff.php" />
+    <rule ref="Generic.PHP.UpperCaseConstant" />
+    <rule ref="Generic.PHP.DeprecatedFunctions" />
+    <rule ref="Squiz.PHP.Eval">
+        <exclude-pattern>*/tests/*</exclude-pattern>
+        <exclude-pattern>*typo3/sysext/install/Classes/CoreUpdates/LocalConfigurationUpdate.php</exclude-pattern>
+    </rule>
+    <rule ref="Squiz.PHP.GlobalKeyword" />
+
+    <!-- Scope -->
+    <rule ref="Squiz.Scope.MemberVarScope" />
+    <rule ref="Squiz.Scope.MethodScope" />
+    <!--<rule ref="../TYPO3SniffPool/Sniffs/Scope/AlwaysReturnSniff.php" />-->
+
+    <!-- Strings -->
+    <rule ref="../TYPO3SniffPool/Sniffs/Strings/UnnecessaryStringConcatSniff.php" />
+    <rule ref="Squiz.Strings.DoubleQuoteUsage" />
+    <rule ref="../TYPO3SniffPool/Sniffs/Strings/ConcatenationSpacingSniff.php" />
+
+    <!-- Whitespace -->
+    <rule ref="Squiz.WhiteSpace.SemicolonSpacing" />
+    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace" />
+    <rule ref="Squiz.WhiteSpace.OperatorSpacing" />
+    <rule ref="../TYPO3SniffPool/Sniffs/WhiteSpace/NoWhitespaceAtInDecrementSniff.php" />
+    <rule ref="../TYPO3SniffPool/Sniffs/WhiteSpace/DisallowSpaceIndentSniff.php" />
```

Comment:
We always use spaces for indentation, looks like this sniff should be excluded instead.

Should we not include the complete `TYPO3SniffPool` and afterwards exclude which we do not need?